src/widget/flex.rs

```diff
@@ -200,7 +205,7 @@ impl<T: Data> Widget<T> for Flex<T> {
             child.widget.set_layout_rect(rect.with_origin(pos));
             major += self.direction.major(rect.size());
         }
-        if flex_sum > 0.0 {
+        if flex_sum > 0.0 && total_major != std::f64::INFINITY {
```

Comment:
Hmm, this seems to be dealing with the case where at least one child is flex but the major dimension is unbounded. In Flutter, that [seems to be an error](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/rendering/flex.dart#L661). Are you sure you want to add special-case logic to deal with it? It might be useful to, at the very least, log a warning when that case is detected.